compiler-core/src/language_server/engine.rs

```diff
@@ -471,6 +471,7 @@ where
 
         // Importable modules
         let import_location = self.first_import_line_in_module(module, line_numbers);
+        let has_any_imports = module.ast.definitions.iter().find_map(get_import).is_none();
```

Comment:
Am I reading this wrong or should this be `is_some`

Reply:
ah, i had my variable called has_no_imports before, lemme fix it

Reply:
I'm a little confused i think... because it works and the tests pass, but my logic seems backwards:
```rust
        let has_any_imports = module.ast.definitions.iter().find_map(get_import).is_none();
```
and
```rust
                let new_lines = if has_any_imports { "\n" } else { "\n\n" };
```

Reply:
It looks like your logic is right for values but reversed for types based on the code and tests in place now